src/main/java/by/exadel/internship/service/impl/InternshipServiceImpl.java

```diff
@@ -131,6 +135,18 @@ public UserInternshipDTO saveInternship(UserInternshipDTO internshipDTO) {
         return userInternshipDTO;
     }
 
+    public boolean addUser(UUID userId, UUID internshipId) {
+        internshipRepository.getOne(internshipId);
+        User user = userRepository.getOne(userId);
+        for (Internship internship : user.getInternships()) {
```

Comment:
I don't like this solution of checking if user is already assigned to the internship. Change please

Reply:
done